web/src/components/AttributeRow/AttributeTitle.tsx

```diff
@@ -0,0 +1,25 @@
+import {Tooltip} from 'antd';
+import Highlighted from '../Highlighted';
+import {Text, TextContainer} from './AttributeRow.styled';
+
+interface IProps {
+  title: string;
+  searchText: string;
+}
+
+export default ({searchText, title}: IProps): JSX.Element => {
+  const textContainer = (
+    <TextContainer>
+      <Text type="secondary">
+        <Highlighted text={title} highlight={searchText} />
+      </Text>
+    </TextContainer>
+  );
+  return title.length > 26 ? (
```

Comment:
This seems to be very specific, isn't there a way to deal with this using CSS?

Reply:
this is because with out current grid column width & font size, when the title kets bigger than 26 character, it gets truncated. css wont work. maybe I can figure out a js solution to infer the column width, but not sure if its worth it